groupGolangPackages.json

```
@@ -0,0 +1,9 @@
+{
+  "packageRules": [
+    {
+      "matchDatasources": ["go"],
+      "matchPackagePatterns": ["aws-sdk-go-v2"],
```

```suggestion
      "matchPackagePatterns": ["^github.com/aws/aws-sdk-go-v2"],
```

would be more specific/appropriate I think.